lua/fspectate/sv_init.lua

```diff
@@ -23,12 +23,18 @@ local function findPlayer(info)
     return nil
 end
 
+local FSpectating = {}
+-- For Lua Refresh
+for k,v in ipairs(player.GetHumans()) do
```

Comment:
`key` argument is useless here, you can replace it with `_`.

Reply:
👍